Tests/ProjectGeneration/Producer.2019.Unity/Assets/PlatformTest.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Linq;
+using UnityEngine;
+
+#if UNITY_WSA
+using System;
+#endif
+
+namespace Microsoft.Build.Unity.ProjectGeneration.Test
+{
+    public enum TestResult
+    {
+        Success,
+        Failure,
+        PlatformNotTested
+    }
+
+    public class PlatformTest
+    {
+        public string Platform => $"{Player}.{Configuration}";
+
+        private string Configuration =>
+#if UNITY_EDITOR
+            "InEditor";
+#else
+            "Player";
+#endif
+
+        private string Player =>
+#if UNITY_ANDROID
+            "Android";
+#elif UNITY_IPHONE
+            "iOS";
+#elif UNITY_WSA
+            "WSA";
+#elif UNITY_STANDALONE
+            "Standalone";
+#endif
+
+#if UNITY_STANDALONE
+        [System.Runtime.InteropServices.DllImport("user32.dll")]
+        private static extern System.IntPtr GetActiveWindow();
```

Comment:
I don't have a significant concern here, but this will break on Mac running the editor.

Reply:
Yeah, and we will have to update with a valid test for that platform then :)